ntp-daemon/src/system.rs

```diff
@@ -272,7 +299,17 @@ impl<C: NtpClock> System<C> {
         self.peers.remove(&index);
     }
 
-    fn handle_spawn(&mut self, peer_address: PeerAddress, addr: SocketAddr) {
+    #[cfg(test)]
+    fn handle_spawn_no_nts(&mut self, peer_address: PeerAddress, addr: SocketAddr) {
+        self.handle_spawn(peer_address, addr, None)
+    }
```

Comment:
Nit: Do we really need this?

Reply:
fully moved it into the `cfg(test)` module now